examples/render_text/app/main.py

```diff
@@ -20,14 +23,46 @@ async def chat_completion(self, request: Request, response: Response):
             # Get the last message content
             content = request.messages[-1].content or ""
 
+            # The image may be returned either as base64 string or as URL
+            # The content specifies the mode of return: 'base64' or 'url'
+            try:
+                command, text = content.split(",", 1)
+                if command not in ["base64", "url"]:
+                    raise DIALException(
+                        message="The command must be either 'base64' or 'url'",
+                        status_code=422,
+                    )
+            except ValueError:
+                raise DIALException(
+                    message="The content must be in the format '(data|url), <text>'",
```

Comment:
I'm not sure but for me `'(data|url),<text>'` (format without a space) looks more stronger